Editor/PropertyDrawers/CinemachineBlendDefinitionPropertyDrawer.cs

```diff
@@ -22,23 +22,22 @@ public override void OnGUI(Rect rect, SerializedProperty property, GUIContent la
             rect.width -= floatFieldWidth + textDimensions.x;
 
             SerializedProperty styleProp = property.FindPropertyRelative(() => myClass.m_Style);
-            if (styleProp.enumValueIndex != (int)CinemachineBlendDefinition.Style.Custom)
-                EditorGUI.PropertyField(rect, styleProp, GUIContent.none);
-            else
+            var r = rect;
+            if (styleProp.enumValueIndex == (int)CinemachineBlendDefinition.Style.Custom)
+                r.width -= 2 * r.height;
+            EditorGUI.PropertyField(r, styleProp, GUIContent.none);
+            if (styleProp.enumValueIndex == (int)CinemachineBlendDefinition.Style.Custom)
```

Comment:
Why no `bool isCustom = m_ShapeProperty.intValue == (int)CinemachineImpulseDefinition.ImpulseShapes.Custom;` like above (in CinemachineImpulseDefinitionPropertyDrawer.cs)?